src/platform/utilities/constants.js

```diff
@@ -1,7 +1,18 @@
+// Relative imports.
+import environment from 'platform/utilities/environment';
+
 // TODO: Rename this to something that makes more sense
 export const requestStates = {
   notCalled: 'not called',
   pending: 'pending',
   succeeded: 'succeeded',
   failed: 'failed',
 };
+
+export const getCernerURL = path => {
```

Comment:
Abstracted this link here based off your suggestion @ncksllvn, let me know if you think there's a better place for it! 🙂

Reply:
I think this is fine place...placing these things are always tricky. The only thing I can think is to create some visibility for it by updating the references in other apps to use the constant, but  I think we have enough changes in this PR already 🙂

Reply:
Thanks for doing this btw!